src/main/java/org/jenkinsci/main/modules/instance_identity/InstanceIdentity.java

```diff
@@ -127,6 +130,15 @@ public RSAPrivateKey getPrivate() {
         return (RSAPrivateKey) keys.getPrivate();
     }
 
+    /**
+     * @since TODO
```

Comment:
Good job to add the `@since` ! :)

FYI the official ordering of tag [from Oracle](https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html#orderoftags). But as it's not really respected in the project, just as a bonus if you are changing things there ;)